src/tracks/InstrumentTrack.cpp

```diff
@@ -121,6 +122,19 @@ InstrumentTrack::InstrumentTrack( TrackContainer* tc ) :
 	}
 
 
+	// Initialize the m_midiCCEnabled variable, but it's actually going to be connected
+	// to a LedButton
+	m_midiCCEnable = new BoolModel(false, nullptr, "Enable/Disable MIDI CC");
+
+	// Initialize the MIDI CC controller models and connect them to the method that processes
+	// the midi cc events
+	for (int i = 0; i < MidiControllerCount; ++i)
+	{
+		m_midiCCModel[i] = new FloatModel(0.0f, 0.0f, 127.0f, 1.0f, NULL, QString("CC Controller ") + QString::number(i));
```

Comment:
The text alone is 115 characters wide. Add 2 tabs at 4 spaces that's 123 characters, at 8 spaces (like GitHub shows for this file) it's 131 characters. Should split it somehow.
```suggestion
		m_midiCCModel[i] = new FloatModel(
			0.0f, 0.0f, 127.0f, 1.0f, NULL, QString("CC Controller ") + QString::number(i));
```

Reply:
I know we usually keep one argument per line, but on this case seemed like an overkill, so I divided the arguments in two lines (also changed NULL to nullptr)